### DISCUSSION TITLE
Roadmap discussion

### DISCUSSION THREAD

**Discussion by IcePigZDB** [CDDSCLab/Weekly-Group-Meeting-Paper-List#32] [Ideas]
Previous discussion: [CDDSCLab/Database-Optimization/issues/7](https://github.com/CDDSCLab/Database-Optimization/issues/7)

hi, @PiotrNewt, @ehds 
@huge-melon init a [repo](https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List) to hold the weekly group meeting topic, paper list and brief summary. There is already one about database transaction shared by him.
* Usually there are more than one paper at a time, so we present it by topic. The topic is organized by shared time currently, as sharing increases, we might consider adding categories by topic and might archived them by semester or year.
* We put the papers list into the summary doc rather than README to make the README concise.
* The summary doc is in Chinese now to reduce written time,  we also hope we can use pure English in the future.

Any suggesion and discussion are appreciate.

**Comment by PiotrNewt** [comment]
It looks great!!! 👍🏼
Nit: what about the previous papers or topics? or just from now on.
